DQMOffline/Trigger/python/MssmHbbMonitor_cfi.py

```diff
@@ -0,0 +1,131 @@
+import FWCore.ParameterSet.Config as cms
+
+mssmHbbBtagTriggerMonitorSL40noMu = cms.EDAnalyzer(
+    "TagAndProbeBtagTriggerMonitor",
+    dirname = cms.string("HLT/Higgs/MssmHbb/Semilep/BtagTrigger/pt40_noMuon"),
```

Comment:
This is semileptonic, just because of the low threshold. In fact the separation is kind of arbitrary for the btag trigger efficiency. For both channels we plan to use all semileptonic and all-hadronic single btag triggers. A dedicated offline analysis with large statistics is needed to spot different efficiencies for jets with and without a muon. In the DQM we should not be able to do that.